src/commands/deploy/deploy.ts

```diff
@@ -624,30 +627,40 @@ const bundleEdgeFunctions = async (options, command: BaseCommand) => {
   })
 }
 
-/**
- *
- * @param {object} config
- * @param {boolean} config.deployToProduction
- * @param {boolean} config.isIntegrationDeploy If the user ran netlify integration:deploy instead of just netlify deploy
- * @param {boolean} config.json If the result should be printed as json message
- * @param {boolean} config.runBuildCommand If the build command should be run
- * @param {object} config.results
- * @returns {void}
- */
-// @ts-expect-error TS(7031) FIXME: Binding element 'deployToProduction' implicitly ha... Remove this comment to see the full error message
-const printResults = ({ deployToProduction, isIntegrationDeploy, json, results, runBuildCommand }) => {
-  const msgData = {
+interface JsonData {
+  site_id: string
+  site_name: string
+  deploy_id: string
+  deploy_url: string
+  logs: string
+  function_logs: string
+  edge_function_logs: string
+  url?: string
+}
+
+const printResults = ({
+  deployToProduction,
+  isIntegrationDeploy,
+  json,
+  results,
+  runBuildCommand,
+}: {
+  deployToProduction: boolean
+  isIntegrationDeploy: boolean
+  json: boolean
+  results: Awaited<ReturnType<typeof prepAndRunDeploy>>
+  runBuildCommand: boolean
+}): void => {
+  const msgData: Record<string, string> = {
     'Build logs': results.logsUrl,
     'Function logs': results.functionLogsUrl,
+    'Edge Function Logs': results.edgeFunctionLogsUrl,
```

Comment:
nit: Edge Functions is only capitalised when referring to the product (plural), not a single function (singular).

```suggestion
    'Edge function Logs': results.edgeFunctionLogsUrl,
```